fe/fe-core/src/main/java/org/apache/doris/analysis/ExpressionFunctions.java

```diff
@@ -96,6 +98,14 @@ public Expr evalExpr(Expr constExpr) {
                     return constExpr;
                 }
             }
+        } else if (constExpr instanceof SysVariableDesc) {
+            try {
+                VariableMgr.fillValue(ConnectContext.get().getSessionVariable(), (SysVariableDesc) constExpr);
+                return ((SysVariableDesc) constExpr).getLiteralExpr();
+            } catch (AnalysisException e) {
```

Comment:
Why not just throw the exception to the user?

Reply:
This method is used for constant folding rules. If it fails, it will directly return to the original expression, and other processes will handle this error.